auditlog/models/rule.py

```diff
@@ -527,34 +509,25 @@ def subscribe(self):
         to view logs on that model.
         """
         act_window_model = self.env['ir.actions.act_window']
-        model_ir_values = self.env['ir.values']
         for rule in self:
             # Create a shortcut to view logs
             domain = "[('model_id', '=', %s), ('res_id', '=', active_id)]" % (
                 rule.model_id.id)
             vals = {
-                'name': _(u"View logs"),
+                'name': _("View logs"),
                 'res_model': 'auditlog.log',
                 'src_model': rule.model_id.model,
```

Comment:
When I test on runbot, I don't get the option to View Logs after subscribing to the logs of a particular model. I believe you need to set binding_model_id to rule.model_id.id here.